.golangci.yml

```diff
@@ -1,6 +1,8 @@
 run:
   timeout: 3m
+  skip-dirs-use-default: false
```

Comment:
The `builtin` directory is in the default list of skipped directories: https://golangci-lint.run/usage/configuration/#run-configuration.